conferences/PyData_Global_2021/tutorial_proposal.md

```diff
@@ -0,0 +1,16 @@
+# PyData2021 tutorial: "How to improve your time series forecast using sktime"
+
+
+## Short description
+Time series are ubiquitous in real-world applications, but often add considerable complications to data science workflows. In this tutorial, you’ll learn about how to apply and extend machine learning techniques to time series. The tutorial will continue the [PyData tutorial from 2020](https://www.youtube.com/watch?v=Wf2naBHRo8Q&t=1240s) with a deeper dive into pipelining, tuning, backtesting. Further, you will learn about the learning problems of time series annotation and time series clustering.
+
+## Long description
+Time series are ubiquitous in real-world applications, but often add considerable complications to data science workflows. What's more, most available machine learning toolboxes (e.g. scikit-learn) are limited to the tabular setting, and cannot easily be applied to time series data. 
+
+In the [PyData tutorial from 2020](https://www.youtube.com/watch?v=Wf2naBHRo8Q&t=1240s) of sktime, you have learned how to apply common machine learning techniques to time series data and how to extend available toolkits.
```

Comment:
I think it's important to also highlight the idea of the unified interface, by pointing to 
* the package we interface (e.g. statsmodels)
* how you can use sktime with sklearn